general_considerations.md

```diff
@@ -167,11 +200,46 @@ assume the DAG does not have constant difficulty so combining difficulties must
 be done correctly. The solution is to identify the heaviest weight linear path
 from genesis to tip, where the difficulties are summed along the path.
 
+[kp]
+
+We don't need to worry about identifying the heaviest weight linear path. We can
+sum up the difficulties of all the shares between epochs, as I describe in my
+proposal. As long as a share has produced a PoW that is valid for the the block
+the miner is working on, they will be rewarded for it.
+
+In other words, I don't think we can actually ever find a block that will be the
+tip of the chain. If there are 500 miners, there will always be 500 shares in
+flight (give or take). We can't wait for them them converge on a tip. Instead,
+the DAG will converge on a tip naturally, as a share matching bitcoin difficulty
+is found and all miners start producing shares that reference that share.
+
+This goes to the point where shares reference other miner's shares to
+build the DAG.
+
+I think we need to break free from the need to find the heaviest weighted tip. I
+need to be convinced we need this when we have a DAG and a natural convergence
+to a tip every time a share matching bitcoin difficulty is found.
+
+[/kp]
```

Comment:
DAG consensus is defined by the heaviest weighted tip.

The "heaviest weight linear path" is only used to do conflict resolution WITHIN the DAG. -- it can decide between two transactions on opposite sides of a diamond graph, for instance.